docs/src/content/docs/quickstart/sql-overview.mdx

```diff
@@ -0,0 +1,636 @@
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+To run your first INSERT transaction, simply enter the following SQL statement into the `psql` prompt, hit return, and you will see:
+
+```sql
+user=> INSERT INTO people (_id, name) VALUES (6, 'fred');
+INSERT 0 0
+```
+
+Note that XTDB doesn't currently return information about the number of rows inserted or modified by a statement when using `psql`.
+  </Card>
+  </TabItem>
+</Tabs>
+
+Here the `people` table has not been defined anywhere beforehand (i.e. there's no explicit `CREATE TABLE` [DDL](https://en.wikipedia.org/wiki/Data_definition_language) statement) - the table is created dynamically during the INSERT statement, along with any supplied columns and inferrable type information.
+
+Each record in XTDB must contain a user-provided `_id` primary key column, but other columns are fully dynamic based on the data that is inserted - meaning each row in XTDB can offer the flexibility of a document in a document database.
+
+The `_` prefix is convention for reserved columns and tables that XTDB handles automatically. For more information see [How XTDB works](/intro/data-model).
+
+### Query for that same row
+
+Querying this data back again is a simple matter of:
+
+<Tabs syncKey="psql">
+  <TabItem label="XT Play" icon="seti:db">
+    <Fiddle magicContext="my-context">
+      <Query q="SELECT * FROM people" />
+    </Fiddle>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+```sql
+user=> SELECT * FROM people;
+ _id | name
+-----+------
+   6 | fred
+(1 row)
+```
+  </Card>
+  </TabItem>
+</Tabs>
+
+### Evolve the table
+
+If we now INSERT another row with a slightly different shape, XTDB will gracefully evolve the schema of the `people` table to reflect the union of all the records it contains:
+
+<Tabs syncKey="psql">
+  <TabItem label="XT Play" icon="seti:db">
+    <Fiddle magicContext="my-context">
+      <Txs systemTime="2024-01-02"
+	   txs={`
+INSERT INTO people (_id, name, likes)
+  VALUES (9, 'bob', ['fishing', 3.14, {dynamic:'data'}])`}/>
+      <Query q="SELECT * FROM people" />
+    </Fiddle>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+```sql
+user=> INSERT INTO people (_id, name, likes)
+  VALUES (9, 'bob', ['fishing', 3.14, {dynamic:'data'}]);
+INSERT 0 0
+user=> SELECT * FROM people;
+ _id |                likes                | name
+-----+-------------------------------------+------
+   6 |                                     | fred
+   9 | ["fishing",3.14,{"dynamic":"data"}] | bob
+(2 rows)
+```
+  <Aside>Due to Postgres' lack of support for polymorphic values and nested structures, the value returned back to the client in the `likes` column is actually JSON-ified string representation of the data we previously inserted, however all original type information is preserved internally should it be needed.</Aside>
+  </Card>
+  </TabItem>
+</Tabs>
+
```

Comment:
I put the note about union types as a tip below, maybe it could also link to the information_schema bit below?
```
:::tip
Behind the scenes XTDB gracefully evolves the schema of `people` to reflect the union of all the types inserted.
:::
```